architecture/effects/random.md

```diff
@@ -113,5 +131,3 @@ At this point, the best way to improve your understanding of commands is just to
 >
 >   - Instead of showing an image of a die face, use the `elm-lang/svg` library to draw it yourself.
 >   - After you have learned about tasks and animation, have the dice flip around randomly before they settle on a final value.
-
```

Comment:
My editor removed these by default, but let me know if you would prefer to keep them and I can add them back